arex-agent-bootstrap/src/main/java/io/arex/agent/bootstrap/cache/TimeCache.java

```diff
@@ -26,9 +31,25 @@ public static void put(long value) {
         }
     }
     public static void remove() {
-        String traceId = TraceContextManager.get();
-        if (traceId != null) {
-            CACHE.remove(traceId);
+        overdueCleanUp();
+    }
+
+    private static void overdueCleanUp() {
+        if (CACHE.mappingCount() > CLEANUP_THRESHOLD && CLEANUP_LOCK.tryLock()) {
+            try {
+                long now = System.nanoTime();
+                for (Map.Entry<String, Pair<Long, Long>> entry : CACHE.entrySet()) {
+                    if (isExpired(entry.getValue().getSecond(), now)) {
+                        CACHE.remove(entry.getKey());
+                    }
+                }
+            } finally {
+                CLEANUP_LOCK.unlock();
+            }
         }
     }
+
+    private static boolean isExpired(long createTime, long now) {
+        return now - createTime > TTL_NANOS;
+    }
```

Comment:
No need to add cleanup code here, recommend to call `TimeCache.remove` in `io.arex.inst.runtime.context.LatencyContextHashMap#overdueCleanUp`